### PR TITLE
add scp command to save work

### PR DESCRIPTION
**Pull request by silvin-lubecki**

### CODE REVIEW

File: exercises/exercise 1/README.md [hunk 1]
```diff
@@ -87,3 +89,11 @@ Status: Downloaded newer image for dapworkshop/workshop:latest
 Now the workshop can really start!
 
 **Don't forget to save on your laptop all your exercises before leaving the workshop**.
+Copy/Paste the `SSH` address and use it to `SCP` all your files:
```
Comment (mikeparker) [inline exercises/exercise 1/README.md]: You should clarify what SCP is for non networking / CLI people. i.e. `SCP` (Secure File Copy)

File: exercises/exercise 1/README.md [hunk 2]
```diff
@@ -28,6 +28,8 @@ https://hub.docker.com
 This workshop uses [Play With Docker](https://labs.play-with-docker.com), so everyone can have the same experience and we don't rely on WiFi to push/pull Gigabytes of images.
 You can still follow the instructions locally with your own engine.
 
+**Don't forget to save on your laptop all your exercises before leaving the workshop**.
```
Comment (mikeparker) [inline exercises/exercise 1/README.md:31]: `You may want to set a reminder to save your exercises locally before leaving the workshop. We will remind you at the end as well.`?

Comment (silvin-lubecki) [reply exercises/exercise 1/README.md:31]: Hope we won't forget to remind them to save 😉

File: exercises/exercise 1/README.md (hunk 1 above)
Comment (silvin-lubecki) [reply exercises/exercise 1/README.md]: Oh sorry missread your comment.

**Pull request merged**